lib/core/src/main/java/io/atlasmap/core/DefaultAtlasCompoundClassLoader.java

```diff
@@ -19,6 +19,7 @@
 import java.util.Collections;
 import java.util.Enumeration;
 import java.util.HashSet;
```

Comment:
It seems unused import checkstyle error is thrown for this one

Reply:
Thanks, fixed d36f617